65.md

```diff
@@ -6,59 +6,138 @@ Relay List Metadata
 
 `draft` `optional`
 
-Defines a replaceable event using `kind:10002` to advertise preferred relays for discovering a user's content and receiving fresh content from others.
+Defines two replaceable events to advertise preferred relays for discovering a user's content and receiving fresh content from others.
 
-The event MUST include a list of `r` tags with relay URIs and a `read` or `write` marker. Relays marked as `read` / `write` are called READ / WRITE relays, respectively. If the marker is omitted, the relay is used for both purposes.
+Relay Usages
+------------
+
+There are three defined usages for a relay:
+
+- `INBOX` relays are for events that the relay list owner will be reading, specifically watching for events that tag them. These do not include DMs (except in the deprecated form of relay list).
+
+- `OUTBOX` relays are for events that the relay list owner has authored and is publishing for public consumption.
+
+- `PRIVATE INBOX` relays are like INBOX relays, except they are for reciept of private messages such as perhaps direct messages (DMs), giftwraps, or the like (but this NIP does not specify what kinds should use relays with this designator).
+
+### When to use which
+
+When posting events (except private ones):
+
+- Clients SHOULD post events to all of their client user's OUTBOX relays,
+- For each p-tag in a posted event, clients SHOULD post the event to at least some of the INBOX relays of the tagged person.
+
+When posting private messages:
+
+- Clients SHOULD post events to at least some (preferably all) of the the p-tag recipient's PRIVATE INBOX relays.
+- Clients SHOULD post a copy to their client user's own PRIVATE INBOX relays.
+- If a user has no PRIVATE INBOX relays, INBOX relays should be used instead.
+
+When seeking events from a user:
+
+- Clients SHOULD seek them from at least some of that user's OUTBOX relays.
+
+When seeking events about a client's user (where the user was tagged):
+
+- Clients SHOULD seek them from all of their user's INBOX relays.
+
+When seeking private messages for a client's user:
+
+- Clients SHOULD seek them from all of their user's PRIVATE INBOX and INBOX relays.
+
+In general, when interacting with a user's own relays, clients should interact with all of them (write to all OUTBOX relays, read from all INBOX and PRIVATE INBOX relays). When interacting with other user's relays, clients need only interact with a subset of them (read from some of their OUTBOX relays, write to some of their INBOX and PRIVATE INBOX relays).
```

Comment:
Since users should merge the results from their OUTBOX relays at the client, each relay might have a different set of records from the user. When interacting with other user's relays, wouldn't clients be missing things if they only select part of the relays?

Reply:
I thought it best to write to all outbox relays, so readers only need to read from some (since they all have the same events). I think that is good for outboxes. Is there still a problem I'm missing regarding outboxes?

Conversely I thought to write to just some inbox relays since the relay user will monitor and read from all of them.

But since your innovation of people reading from other people's inboxes, that kinks it up. Since they would now have to read from all of them.

Reply:
To make it real: to seek replies to a note from user A, I would use all of A's INBOX servers.

Reply:
Why don't we say writers write to all, readers read from some, in all cases?  That makes it easier.

Reply:
I think it works. It would be nice if users could setup a mirroring service between the relays in the same category.

Reply:
To make it real 2: if you know I am following you (I post my follow list in your Inbox, because you were cited in it), why don't you send your posts to each follower's INBOX relay as if they were directly cited? Then clients don't need to connect to your relay. 

A DVM, or a broadcasting service, can do that for clients.